src/towncrier/_settings/load.py

```diff
@@ -162,23 +173,7 @@ def parse_toml(base_path: str, config: Mapping[str, Any]) -> Config:
             f"The template file '{template}' does not exist.",
             failing_option="template",
         )
+    parsed_data["template"] = template
 
-    return Config(
-        package=config.get("package", ""),
-        package_dir=config.get("package_dir", "."),
-        single_file=single_file,
-        filename=config.get("filename", "NEWS.rst"),
-        directory=config.get("directory"),
-        version=config.get("version"),
-        name=config.get("name"),
-        sections=sections,
-        types=types,
-        template=template,
-        start_string=config.get("start_string", _start_string),
-        title_format=config.get("title_format", _title_format),
-        issue_format=config.get("issue_format"),
-        underlines=config.get("underlines", _underlines),
-        wrap=wrap,
-        all_bullets=all_bullets,
-        orphan_prefix=config.get("orphan_prefix", "+"),
-    )
+    # Return the parsed config.
+    return Config(**parsed_data)
```

Comment:
I'm gonna be honest and say I'm not a fan of this function, but I guess to make it nicer, we'd have to move to something like jsonschema so I'm not gonna hold this PR over it. It's difficult to gauge it's correctness, due to many changes, but I'm gonna trust the tests…

Reply:
Yeah, sorry -- I know it's a big change but it is at least in a more logical and readable order now. Also, it's far better that we're using Config defaults more correctly.